bundles/org.openhab.binding.dwdpollenflug/src/main/java/org/openhab/binding/dwdpollenflug/internal/config/DWDPollenflugBridgeConfiguration.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dwdpollenflug.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.dwdpollenflug.internal.handler.DWDPollenflugBridgeHandler;
+
+/**
+ * Configuration for the {@link DWDPollenflugBridgeHandler}
+ *
+ * @author Johannes DerOetzi Ott - Initial contribution
+ */
+@NonNullByDefault
+public class DWDPollenflugBridgeConfiguration {
+    private int refresh;

```

Comment:
But in Java logic this is not the same, a private field is although settable from outside the class if I want to prevent this I think this should be a private with a getter.

Reply:
True, but who should do that in a private member of a class? The configuration is not available outside of the handler.

Reply:
Changed! Please check my fix!